demos/weather_forecast/docker-compose-jaeger.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
I presume this file is copied, and did we test this after the new folder structure?

Reply:
yes, this was tested after the new folder structure with jaeger ui and everything